conans/test/functional/toolchains/cmake/cmakedeps/test_cmakedeps.py

```diff
@@ -199,7 +201,7 @@ def package_info(self):
 
     conanfile = textwrap.dedent("""
         [requires]
-        test/0.1
+        test/0.1.1
```

Comment:
perhaps not entirely necessary, but are we testing the case where we have either `SameMajorVersion` or `SameMinorVersion` and the requirement does not satisfy this? (perhaps it's covered by a test not in the PR, since that's the default behaviour prior to the PR)

Reply:
I can add some test to cover this, I was trying to minimize the addition of new tests that call CMake, as they are very slow.

Reply:
hehe sorry about this!